### ISSUE TITLE
Standby graph

### ISSUE THREAD

**Issue by rubenbaetens** [opengridcc/opengrid#100]
I'll update the standby graphs of opengrid.be to look better, and include an external reference for it. That is, not only show a boxplot of the current fluksos but also add e.g. a EU-27 average as reference.

**Comment by rubenbaetens** [comment]
Where are these plot functions actually located ? I can't find them anywhere: 
- `plotting.py` only contains the `carpet` and `fanchart` function, while 
- `analysis.py` only contains the `daily_min` and `daily_max` function.
and these seem to me the only two place where they should logically be located.

**Comment by rubenbaetens** [comment]
Related, I found parts of the code in `Notebooks/Benchmark_night-time_standy_electricity_consumption` and `Notebooks/job_SyncAndRenderGraphs`. The latter however denotes:
```
hp.save('/usr/local/src/opengrid/scripts/hp_anonymous.pkl')
```
while there is nu such directory as `opengrid/scripts` in `develop`.

**Comment by saroele** [comment]
We should extract useful and generic functions from this script and add this to the library (in analysis, or plotting).  Also, we should use the caching mechanism to create graphs with longer time spans. 

I hope to attack this by the end of the week.  Maybe you should wait until the graphs work again, and then start from there? thanks for your help :-)

**Comment by saroele** [comment]
I have updated the notebook: it's much better to read as it now uses the caching function.  The code is much easier.  I have increased the length of the graphs (specifically the 'vertical' one as it is suited for longer graphs). 

The graphs appear on www.opengrid.be again, so you can look at the result without running anything.

**Comment by WolfBerwouts** [comment]
Well done!

**Comment by wouterh** [comment]
@saroele What's the x-axis on the vertical one? It seems to be days, but then it's from a long time ago. And I would expect 1 dot (for your standby power) per day ...

**Comment by saroele** [comment]
Yes, it's days, one dot per day.  Thanks to the caching, it's now easy to make overviews (of daily results of some analysis) over a longer time.

**Comment by wouterh** [comment]
@saroele the labels seem to be off ... Labels are now 29/10 - > 04/11, which is 7 days, but there far more dots...

**Comment by saroele** [comment]
Indeed,  they are wrong!  I'll fix it.  Thanks for remarking :-)
Op 5 feb. 2016 13:47 schreef "Wouter Horré" <notifications@github.com>:

**Comment by saroele** [comment]
Fixed the dates.  The graph may not be the final one, but at least it is correct now. 
I reduced the number of days to plot from 100 to 40.

**Comment by rubenbaetens** [comment]
All standby-graphs on the same page have a different y-axis range (e.g. see http://opengrid.be/sensor/53b1eb0479c83dee927fff10b0cb0fe6 with a [0-200], [0-500] and [0-1200] axis). From a dataviz approach, it's better to have them all equal.
